### PR TITLE
Allow for custom corpus in LSA primitive

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Update LSA to allow for users to optionally specify a corpus to use. If not specified, will default to the nltk Gutenberg corpus.

### CODE REVIEW

File: nlp_primitives/lsa.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ class LSA(TransformPrimitive):
 
         If a string is missing, return `NaN`.
 
+        Note: If a small custom corpus is used, the output of the primitive may vary
+        depending on the computer architecture being used (linux, macos, windows). This
```
Comment (gsheni) [inline nlp_primitives/lsa.py:30]:
```suggestion
        depending on the computer architecture being used (Linux, MacOS, Windows). This
```

**Review by gsheni — DISMISSED** [review]
lgtm

**Review by rwedge — COMMENTED** [review]
should we add a test for a non-randomize algorithm / bad input?

**Comment by thehomebrewnerd** [comment]
> should we add a test for a non-randomize algorithm / bad input?

I can add a quick test for that and code to handle.

**Review by rwedge — APPROVED** [review]

**Pull request merged**